f/fcs/fn_fcsInit.sqf

```diff
@@ -81,9 +81,13 @@ _vehicle setVariable ["f_var_fcs_hasEH",true];
 		params ["_target", "_caller", "_actionId", "_arguments"];
 		// Clear failure states
 		_target setVariable ["f_var_fcs_failure",false,true];
-		_target disableNVGEquipment false;
+		// Retrieve saved info on the previous TI/NV equipment state
+		private _equipInfo = _target getVariable ["f_var_fcs_equipInfo",[false,false]];
+		if !(_equipInfo#0) then {
+			_target disableNVGEquipment false;
+		};
```

Comment:
```suggestion
		_target disableNVGEquipment (_equipInfo#0);
```
Minor style suggestion: since `disableNVG(/TI)Equipment` takes a boolean, the result could be used directly rather than using an if.